sdk_container/src/third_party/coreos-overlay/coreos-base/oem-ami/files/manglefs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rootfs="${1}"
+
+cat > "${rootfs}/usr/lib/systemd/system/setup-oem.service" <<-'EOF'
+[Unit]
+Description=Setup OEM
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=/usr/bin/cp /usr/share/amazon/ssm/amazon-ssm-agent.json /etc/amazon/ssm/amazon-ssm-agent.json.template
+ExecStartPre=/usr/bin/cp /usr/share/amazon/ssm/seelog.xml /etc/amazon/ssm/seelog_unix.xml.template
+ExecStartPre=/usr/bin/cp /usr/share/amazon/eks/bootstrap.sh /etc/eks/bootstrap.sh
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+mkdir -p "${rootfs}/usr/lib/systemd/system/multi-user.target.d"
+{ echo "[Unit]"; echo "Upholds=amazon-ssm-agent.service coreos-metadata-sshkeys@.service setup-oem.service"; } > "${rootfs}/usr/lib/systemd/system/multi-user.target.d/10-oem-ami.conf"
```

Comment:
Enabling `coreos-metadata-sshkeys@.service` is something we should do in the base image.